src/domTree.js

```diff
@@ -68,13 +68,14 @@ class span implements CombinableDomNode {
         classes?: string[],
         children?: DomChildNode[],
         options?: Options,
+        style?: {[string]: string},
     ) {
         this.classes = classes || [];
         this.children = children || [];
         this.height = 0;
         this.depth = 0;
         this.maxFontSize = 0;
-        this.style = {};
+        this.style = style || {};
```

Comment:
`this.style` is mutated below, so it seems dangerous to use the passed in style object without copying it-it probably works now, but could be a source of difficult bugs the the calling code changes to reuse the style object. 

Does this project have babel set up so that you can use `{...style}` here?

The rest of my comments are just nits, but for this one I'm actually going to request changes...

Reply:
Whoops, missed the one comment I actually need to address.  Fixing now. :)